x-pack/plugins/cases/server/services/user_actions/index.ts

```diff
@@ -249,6 +249,8 @@ export class CaseUserActionService {
         filters: [
           UserActionTypes.comment,
           UserActionTypes.description,
+          UserActionTypes.severity,
+          UserActionTypes.status,
```

Comment:
@jonathan-buttner I see you implemented this logic. Here we check if any of these fields have updated since the last push date. I added `severity` and `status`. I thought I'd try to do this for `.cases-webhook` type connector only, but I don't think I can access that here. Do you know a way, or is it ok to add these properties for all connectors?

Reply:
hmm this logic is used by the algorithm to determine if a case is flagged as needing to be pushed for a particular connector.

Like you said if we add these fields here then all connectors will be affected even if they don't push those fields.

I think up to this point all case connectors support the same fields which is why we got away with common functionality with all of them. I think we'd need a more complicated solution if we want connectors to be marked as needing to push based on different fields than other connectors.

I've actually transitioned to the ML team so I'm going to defer to the cases team for this decision

@elastic/response-ops-cases what do you think?

One solution might be that we check which connector is currently selected and dynamically determine which filter fields to use based on its type. So for cases-webhook we check all these fields, for other connectors we only do comment, description, title, tags.

Reply:
I've got this fixed now, resolving. will post screenshot in PR description